react/components/Payment/AddittionalInfo.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FunctionComponent } from 'react'
+import { InjectedIntlProps, injectIntl } from 'react-intl'
+
+import Tooltip from '../../Icons/Tooltip'
+
+interface Props {
+  paymentId: string
+  transactionId: string
+  showTooltip?: boolean
+}
+
+const AdditionalInfo: FunctionComponent<Props & InjectedIntlProps> = ({
+  paymentId,
+  transactionId,
+  showTooltip,
+  intl,
+}) => {
+  return (
+    <div className="flex flex-column">
+      {showTooltip && <Tooltip />}
+      <div className="bg-base--inverted pa4 br2">
+        <p className="white tc">
```

Comment:
Use css token for text color as well.

Reply:
The best token I found for `#ffffff` was `c-on-base--inverted`

Reply:
Makes sense since you're using `bg-base--inverted` in the parent element.